### PR TITLE
Some Libultra data and decompilation work

### PR DESCRIPTION
**Pull request by JoshDuMan**

### CODE REVIEW

**Review by ethteck — CHANGES_REQUESTED** [review]
Awesome work! - just a couple style things

File: src/os/osSetThreadPri.c [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+{
+    OSThread *next;
+    OSPri priority;
+};
+
+struct __osThreadTail __osThreadTail = {0, -1};
+OSThread *__osRunQueue = (OSThread *)&__osThreadTail;
+OSThread *__osActiveQueue = (OSThread *)&__osThreadTail;
+OSThread *__osRunningThread = {0};
+OSThread *__osFaultedThread = {0};
```
[the remark this answers is not in the document]
Comment (nanaian) [reply src/os/osSetThreadPri.c:13]: Same note on pointer times goes for the casts on line 10 and 11, just do `(OSThread*)&__osThreadTail;`

File: src/os/osSetThreadPri.c [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+#ifdef MOVE_ISSUE
```
[the remark this answers is not in the document]
Comment (ethteck) [reply src/os/osSetThreadPri.c:15]: We do, and I told him to

**Comment by JoshDuMan** [comment]
I believe all issues have been addressed. Most were carry-overs from Libultra I thought would be fixed with format.sh

**Pull request merged**